src/Common/src/Interop/Unix/System.Native/Interop.MessageHeader.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        internal unsafe struct IOVector
+        {
+            public byte* Base;
+            public UIntPtr Count;
```

Comment:
I don't understand why this is UIntPtr.  Shouldn't it be an int, uint, long, or ulong?  It's not really a pointer/handle, right?